src/signalcorr.jl

```diff
@@ -87,6 +87,36 @@ end
 
 autocov{T<:Real}(x::VecOrMat{T}; demean::Bool=true) = autocov(x, default_autolags(size(x,1)); demean=demean)
 
+"""
+    autocov(x, lags, demean)
+
+### Args:
+* `x`: Input `Vector` or `Matrix` to compute the autocovariance of. 
+* `lags`: Specifies the lags at which the autocovariance of `x` is copmuted.
+* `demean`: A `Bool` type variable, this is a named paramter. 
+
+Compute auto-covariance of `x` at specified lags. If `x` is a vector, it returns a vector of the same length of lags. If `x` is a matrix, it returns a matrix of size `(length(lags), size(x,2))`, where each column in the result corresponding to a column in `x`.
+
+Here, demean is a keyword argument (default value is true), which means that the function will subtract each `x` from its mean before computing the results. Otherwise, `x` is considered as having been centered.
```

Comment:
"demean is a keyword argument (default value is true)," won't be needed if you add this information to the signature.

I also think "subtract each `x` from its mean" should be "subtract its mean from each column in `x` (i.e. center it)".